lib/login-providers/browser.js

```diff
@@ -76,7 +76,8 @@ class BrowserLoginProvider extends LoginProviderBase {
             reject(body.error_description);
             return;
           }
-          resolve(super.storeJWT(body.id_token));
+          // Must use this.storeJWT instead of super.storeJWT to support bug in Node 4.
+          resolve(this.storeJWT(body.id_token));
```

Comment:
This was breaking tests: http://stackoverflow.com/questions/32943776/using-super-within-an-arrow-function-within-an-arrow-function-within-a-method